Documentation/Signoffs/ResistanceSystem.md

```diff
@@ -1,76 +1,138 @@
-# Team 4 – Mario Kart & Ride Replay Simulator
+Team 4 – Mario Kart & Ride Replay Simulator
 
 Team Members: Ray Durlin, Blake Pickett, Tyler Chittum, Benjamin Reed, and Sage Mooneyham
 
 ## Detail Design: Resistance System ##
 
 
-_Function of the Subsystem:_ 
+### Function of the Subsystem:
 
-The function of the resistance system is to control and vary the resistance felt by the user of the simulator. The resistance system contains an H-Bridge microcontroller that receives a wirelessly input resistance level value from the RRS processing subsystem in the form of a numbered value ranging from 0 to 85. The H-bridge supplies either a positive or negative voltage to the linear actuator to control the direction of the actuator's motion, and the on-board microcontroller controls the duty cycle of the applied voltage to modulate speed. The microcontroller takes an analog input from the linear actuator to determine its current position, as well as an external digital input (via the Bluetooth module) to determine the calculated set point which the actuator should try to reach. The linear motor is attached to an array of N52 neodymium magnets. The linear motor is to vary the array of magnets perpendicular to the simulator’s rear conductive flywheel. As the magnet array is varied perpendicular to the conductive flywheel, eddy currents are generated by the primary magnetic induction formed at the conductor [1]. As the eddy currents form, braking torque is also generated due to the opposing magnetic field. This torque produced in the flywheel is then transferred to the rear bike wheel. The braking torque functions as the resistance felt by the user. The braking torque ranges from 0 to 28.9 N∙m based on the input resistance level received by the H-Bridge controller. 28.9 N∙m is the greatest amount of torque available given the user is pedaling at the minimum speed.
+The function of the resistance system is to control and vary the resistance felt by the user of the Mario Kart Bike, hereafter referred to as the bike. 
 
-**_Constraints:_**
+•	The H-Bridge microcontroller receives a wireless input from the RRS processing subsystem in the form of a numbered value ranging from 0 to 85, which represents the resistance level to be experienced by the user.
```

Comment:
This seems like it might not consider the speed the flywheel is turning in choosing the distance.